modules/core/server/views/layout.server.view.html

```diff
@@ -17,7 +17,12 @@
     <meta name="fragment" content="!" />
     {% for cssFile in cssFiles %}
     <link rel="stylesheet" href="/{{cssFile}}?c={{appSettings.commit}}" />
-    {% endfor %} {% if env === 'development' %} {# in dev load js in head, as it
+    {% endfor %} {# Sentry DSN - make sure this is above scripts #} {% if
+    sentry.enabled === true %}
+    <script>
+      window.SENTRY_DSN = '{{ sentry.options.dsn }}';
```

Comment:
We could also pass this via Webpack config — I'm not tied to the way or the other tho so fine as is.

Reply:
I actually did it like that originally, but then realised we may not have the sentry configuration available at the time the webpack build is done (esp. in this new approach where the build is done in CI).

I think it would be confusing if some of the stuff in `config/` would update immediately and some only after a webpack rebuild.

Given there is a mechanism to pass config from server to client I think it's nice to expand on this (... and could be made a more _proper_ and _general_ mechanism in a future enhancement!)